tool/travis.sh

```diff
@@ -40,7 +40,7 @@ html_tests="$(find test -type f -name "html_*_test.dart")"
 
 for test in ${html_tests[@]}
 do
-  pub run test -r expanded -p chrome "$test"
+  pub run test -r expanded -p content-shell "$test"
```

Comment:
We need to ditch usage of content shell soon. Should use chrome – it runs headless by default...